evennia/utils/evmenu.py

```diff
@@ -64,10 +68,37 @@ def another_node(caller, input_string, **kwargs):
 The return values must be given in the above order, but each can be
 returned as None as well. If the options are returned as None, the
 menu is immediately exited and the default "look" command is called.
-
-    text (str, tuple or None): Text shown at this node. If a tuple, the
-        second element in the tuple is a help text to display at this
-        node when the user enters the menu help command there.
+    text (str, tuple, dict or None): Text shown at this node. If a dict
+        it can also contain various formatting instructions. In such a 
+        case the followng keys are already supported:        
+            - 'text' (str): The text of the node.
+            - 'help' (str): The help text of the node.
+            - 'footer' (str): Footer text for the node.
```

Comment:
Why no `header` text when we are at it?

Reply:
Mostly because I didn't think when I was writing this how ``header`` was any different than something at the beginning of the textfield. Now that you've brought it up I can see how some formats might want to separate it for use as a title (above the top bar line) or something like that.

Reply:
I went ahead and added a ``header`` element. This is too useful for things such as a title at the top of the menu which can be centered by the node formatter if desired.